jujugui/static/gui/src/app/app.js

```diff
@@ -908,10 +908,12 @@ YUI.add('juju-gui', function(Y) {
           controllerAPI, bakery)}
       />);
 
-      let logoutUrl = '/logout';
+      let logoutUrl = 'logout';
+      let baseUrl = '/';
       if (window.juju_config.baseUrl) {
-        logoutUrl = window.juju_config.baseUrl + logoutUrl;
+        baseUrl = window.juju_config.baseURL;
       }
+      logoutUrl = baseUrl + logoutUrl;
```

Comment:
This could just be:
```javascript
const logoutUrl = `${(window.juju_config.baseURL || '/')}logout`;
```
or something similarly snappy if you so desired.